pydis_site/apps/content/resources/guides/pydis-guides/contributing/sir-lancebot/env-var-reference.md

```diff
@@ -6,69 +6,59 @@ toc: 2
-| `CHANNEL_REDDIT` | ID of the `#reddit` channel |
+| Environment Variable               | Description                                                                                |
+|------------------------------------|--------------------------------------------------------------------------------------------|
+| `CLIENT_TOKEN`                     | Bot Token from the [Discord developer portal](https://discord.com/developers/applications) |
+| `CLIENT_GUILD`                     | ID of the Discord Server                                                                   |
+| `ROLES_ADMINS`                     | ID of the role `@Admins`                                                                   |
+| `ROLES_HELPERS`                    | ID of the role `@Helpers`                                                                  |
+| `CHANNELS_ANNOUNCEMENTS`           | ID of the `#announcements` channel                                                         |
+| `CHANNELS_DEVLOG`                  | ID of the `#dev-log` channel                                                               |
+| `CHANNELS_SIR_LANCEBOT_PLAYGROUND` | ID of the `#sir-lancebot-commands` channel                                                 |
+| `CHANNELS_REDDIT`                  | ID of the `#reddit` channel                                                                |
 
 ---
 ## Debug Variables
 Additionally, you may find the following environment variables useful during development:
 
-| Environment Variable | Description |
-| -------- | -------- |
-| `BOT_DEBUG` | Debug mode of the bot | False |
-| `PREFIX` | The bot's invocation prefix | `.` |
-| `CYCLE_FREQUENCY` | Amount of days between cycling server icon | 3 |
-| `MONTH_OVERRIDE` | Integer in range `[0, 12]`, overrides current month w.r.t. seasonal decorators |
-| `REDIS_HOST` | The address to connect to for the Redis database. |
-| `REDIS_PORT` | |
-| `REDIS_PASSWORD` | |
-| `USE_FAKEREDIS` | If the FakeRedis module should be used. Set this to true if you don't have a Redis database setup. |
-| `BOT_SENTRY_DSN` | The DSN of the sentry monitor. |
-| `TRASHCAN_EMOJI` | The full emoji to use for the trashcan. Format should be like the output of sending `\:emoji:` on discord. |
+| Environment Variable       | Description                                                                                                |
+|----------------------------|------------------------------------------------------------------------------------------------------------|
+| `CLIENT_DEBUG`             | Debug mode of the bot                                                                                      | False |
+| `CLIENT_PREFIX`            | The bot's invocation prefix                                                                                | `.` |
+| `BRANDING_CYCLE_FREQUENCY` | Amount of days between cycling server icon                                                                 | 3 |
```

Comment:
The third column here seems to be ignored when rendering. They should probably either be removed, or given a header and filled in for some other values where helpful (e.g. `REDIS_USE_FAKEREDIS`). (Also, the default for `CLIENT_DEBUG` is `True`)

Reply:
I didn't even see the third column since it doesn't fit on my screen.
I'll just remove them.